### PR TITLE
Update style.scss

### LINKED ISSUE #37: Make resource tiles same height
On several pages, the height of tiles currently varies based on the amount of text in each one.  Find a way to make the tiles equal height.

### PR DESCRIPTION
**Pull request by jedington**
#37

### CODE REVIEW

**Comment by theackstack** [comment]
Did some research into this problem a few weeks ago and saw that there are several ways to solve it, but it looked like there were some tradeoffs and challenges with each method.

It looks like your changes did make the resource tiles the same height, so we're on the right track!  Currently when I run your branch, there are some scrollbars that pop up on some of the tiles with longer text, depending on the height of the screen:

issue-37-pr-review-1

Is there a way to fix this?  The most ideal result would make all the tiles the same height as the tallest one in the row, so that no scrollbars are needed...I envisioned something like the following:

issue-37-pr-review-2

Do you know if it's possible to do this using only CSS?

**Comment by jedington** [comment]
With the current method, we can just make the boxes taller, but at some point they can be too tall. The other option is to 'box' them, but the formatting is all over the place and would require tweaking those lines. I'll try the box method and see if it's a better option (I did almost go with that, but I noticed it would require alot more work to function).

**Comment by jedington** [comment]
Here's a solid solution I found after scrubbing for guides the past 2 hours:
https://medium.com/wdstack/bootstrap-equal-height-columns-d07bc934eb27
I was looking between inline options as well as flex and even grid methods.

Best way (for responsive design in this case) is to use 'flexbox':
With the addition of a `flex-wrap: wrap` and then a `flex-direction: column`.

```
.resource-row {
  display: flex;
  flex-wrap: wrap;
}
.resource-div {
 display: flex;
 flex-direction: column;
 width: 44%;
 padding: 13px;
 margin: 4px 4px 4px 4px;
 background-color: white;
 text-align: center;
 box-shadow: 0px 0px 10px #888
}
```

File: assets/css/style.scss [hunk 1]
```diff
@@ -247,18 +247,23 @@ Divs
...
-  padding: 20px;
-  margin: 10px;
-  background-color: white;
-  display: inline-block;
-  text-align: center;
-  vertical-align: top;
-  box-shadow: 0px 0px 10px #888888;
+ display: flex;
+ flex-direction: column;
+ width: 44%;
```
Comment (theackstack) [inline assets/css/style.scss]: Re-set width to concrete size - this enables the "stacking" behavior as the width of the screen shrinks as opposed to the tiles shrinking into very tall, skinny columns.  I've found that 240px seems to be the sweet spot where the tiles are aesthetically pleasing and fit the content well.

File: assets/css/style.scss [hunk 2]
```diff
@@ -247,18 +247,23 @@ Divs
...
+ padding: 13px;
+ margin: 4px 4px 4px 4px;
```
Comment (theackstack) [inline assets/css/style.scss]: Let's keep padding: 20px and margin: 10px for simplicity and to allow for a little more space between tiles and content

File: assets/css/style.scss [hunk 3]
```diff
@@ -247,18 +247,23 @@ Divs
...
+ background-color: white;
+ text-align: center;
+ box-shadow: 0px 0px 10px #888
```
Comment (theackstack) [inline assets/css/style.scss]: Nit: hex value got changed and missing semicolon

File: assets/css/style.scss [hunk 4]
```diff
@@ -247,18 +247,23 @@ Divs
...
+}
+
+.resource-div:hover {
+  transform: scale(.98);
+  box-shadow: 0px 0px 30px #ebebeb
```
Comment (theackstack) [inline assets/css/style.scss]: Nit: missing semicolon

File: assets/css/style.scss [hunk 5]
```diff
@@ -247,18 +247,23 @@ Divs
 *******************************************************************************/
 
 .resource-row {
-  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
```
Comment (theackstack) [inline assets/css/style.scss:251]: Fantastic, it looks like this works beautifully!

File: assets/css/style.scss [hunk 6]
```diff
@@ -247,18 +247,23 @@ Divs
...
+ flex-direction: column;
+ width: 44%;
+ padding: 13px;
+ margin: 4px 4px 4px 4px;
+ background-color: white;
+ text-align: center;
+ box-shadow: 0px 0px 10px #888
+}
+
+.resource-div:hover {
```
Comment (theackstack) [inline assets/css/style.scss:264]: I like the hover animation on the tile as a whole.  This was a great addition.

**Review by theackstack — CHANGES_REQUESTED** [review]
Super awesome that you were able to figure this out!  These tiles are much more aesthetically pleasing as a result.

Just a few small changes based on my personal preference for sizing and spacing and the look/feel of the site.  Then we should be good to go!

File: assets/css/style.scss (hunk 1 above)
Comment (jedington) [reply assets/css/style.scss]: I personally like %'s for different size devices in mind, but there's a downside with much wider screens I suppose.

File: assets/css/style.scss (hunk 4 above)
Comment (jedington) [reply assets/css/style.scss]: Readded. Browser dev tools remove the semicolon for each ending line before a curly bracket.

File: assets/css/style.scss (hunk 3 above)
Comment (jedington) [reply assets/css/style.scss]: reverted

File: assets/css/style.scss (hunk 2 above)
Comment (jedington) [reply assets/css/style.scss]: noted, reverted

**Comment by jedington** [comment]
Changes made according to comments.

File: assets/css/style.scss (hunk 1 above)
Comment (theackstack) [reply assets/css/style.scss]: I agree with you that there are some benefits to using percentages.  In this case, with a % value here, when I shrink the screen width, the columns all become very skinny and end up with one word per line and some overflow.  Personally I think it's a better user experience to have them start stacking instead as the screen width gets shorter.

Are you observing a different behavior on your end with the percentages?

File: assets/css/style.scss (hunk 4 above)
Comment (theackstack) [reply assets/css/style.scss]: Great insight, thanks

**Review by theackstack — APPROVED** [review]
LGTM

**Pull request merged**